### ISSUE TITLE
Investigate if we should use ~30 min cache duration if we have no etag/mod headers

### ISSUE THREAD

**Issue by wolfv** [mamba-org/mamba#1504]
Some anaconda.org channels have no `etag` / `mod` headers and thus there is always a request to check if the data has changed (e.g. the case for the `robostack` channel). Maybe in this case we should always default to 30 min cache duration.

**Comment by jonashaag** [comment]
Related, I've done some scraping of conda-forge over the past day and it never seems to update more frequently than 1x per hour (as discussed a few days ago), while always reporting an `Expires` header that's 20 minutes in the future. Maybe we want to override the header for that particular case.

**Comment by wolfv** [comment]
Yes, channels that are behind a CDN (conda-forge, defaults, bioconda) behave differently. 

Additionally they have an ETag header that is sent with a next request. Through that etag header, the server can check wether it should start sending new content (it's basically a MD5 sum of the content).
